src/main/java/dk/kb/kaltura/client/DsKalturaClient.java

```diff
@@ -73,15 +73,17 @@ public class DsKalturaClient {
      *
      * @throws IOException  If session could not be created at Kaltura
      */
-    public DsKalturaClient(String kalturaUrl, String userId, int partnerId, String adminSecret, long sessionKeepAliveSeconds) throws IOException {        
+    public DsKalturaClient(String kalturaUrl, String userId, int partnerId, String token, String tokenId, String adminSecret, long sessionKeepAliveSeconds) throws IOException {
```

Comment:
Missing javadoc for  token. Also method explain one of token or adminsecret must not be null.

Reply:
javadoc updated